### DISCUSSION TITLE
Subscribing to threads &  conversations

### DISCUSSION THREAD

**Discussion by yoshimo** [mastodon/mastodon#21661] [Q&A]
Often I find that a certain post sparks an interesting conversation.
I'd like to keep tracks on it and be notified  when people reply to the post or a sub branch of the conversation tree.
This would be nice to have as optin for every post I mark as favourite or reply to.
Is that  something the api allows already?

**Comment by trwnh** [comment]
#8153

**Comment by shaedrich** [comment]
#7969
